common/src/test/java/io/honeycomb/opentelemetry/DistroMetadataTest.java

```diff
@@ -0,0 +1,18 @@
+package io.honeycomb.opentelemetry;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DistroMetadataTest {
+
+    @Test
+    public void test_distroMetadata_returns_expected_values() {
+        Map<String, String> metadata = DistroMetadata.getMetadata();
+        Assertions.assertNotNull(metadata.containsKey("honeycomb.distro.version"));
+        Assertions.assertEquals(metadata.get("honeycomb.distro.version"), "0.6.1");
```

Comment:
I am generally in favor of explicit string comparisons in tests, but maybe this could be a project version lookup so that the tests don't need to be changed for each release.

Reply:
After a face-to-face, I think @MikeGoldsmith and I are in favor of making assertions that the class constant(s) appear in the string map and care less about their values here in the unit test. We can make an assertion that the actual value declared in build.gradle appears by checking in the smoke tests.